src/controlnet_aux/dwpose/wholebody.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import numpy as np
+from . import util
+import cv2
+import mmcv
+import torch
+import matplotlib.pyplot as plt
+from mmpose.apis import inference_topdown
+from mmpose.apis import init_model as init_pose_estimator
+from mmpose.evaluation.functional import nms
+from mmpose.utils import adapt_mmdet_pipeline
+from mmpose.structures import merge_data_samples
+
+from mmdet.apis import inference_detector, init_detector
+
+
+class Wholebody:
+    def __init__(self, 
+                 det_config=None, det_ckpt=None, 
+                 pose_config=None, pose_ckpt=None,
+                 device="cpu"):
+                
+        if det_ckpt is None:
+            det_ckpt = 'https://download.openmmlab.com/mmdetection/v2.0/yolox/yolox_l_8x8_300e_coco/yolox_l_8x8_300e_coco_20211126_140236-d3bd2b23.pth'
+        
+        if pose_ckpt is None:
+            pose_ckpt = "https://huggingface.co/wanghaofan/dw-ll_ucoco_384/resolve/main/dw-ll_ucoco_384.pth"
+        
```

Comment:
How does this handle when the config variables are None?

Reply:
Fixed.